### PR TITLE
Added support for OnPushOpened in the listener

### PR DESCRIPTION
**Pull request by devinsmythe**
Added support for OnPushOpened in the listener.

### CODE REVIEW

**Comment by rlepinski** [comment]
👍

**Comment by BrianBatchelder** [comment]
retest this please

File: Assets/UrbanAirship/Platforms/UAirship.cs [hunk 1]
```diff
@@ -293,6 +303,19 @@ private class UrbanAirshipListener : MonoBehaviour {
 				}
 			}
 
+			void OnPushOpened (string payload) {
```
Comment (BrianBatchelder) [inline Assets/UrbanAirship/Platforms/UAirship.cs]: weird indenting after here?

Comment (rlepinski) [reply Assets/UrbanAirship/Platforms/UAirship.cs]: No its in an inner class

Comment (BrianBatchelder) [reply Assets/UrbanAirship/Platforms/UAirship.cs]: :thumbs:

**Comment by BrianBatchelder** [comment]
👍

**Pull request merged**